.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: minimal
 
-dist: xenial
+dist: bionic
```

Comment:
Why is this change needed?

Reply:
Ubuntu 16.04 LTS reaches End Of Life (EOL). Please refer to https://ubuntustudio.org/2019/04/ubuntu-studio-16-04-lts-reaches-end-of-life-eol/.

I change it to ubuntu 18.04.